Library/Homebrew/keg.rb

```diff
@@ -88,6 +88,10 @@ def to_s
     mime-info pixmaps sounds postgresql
   ].freeze
 
+  ALL_TOP_LEVEL_PATHS = ALL_TOP_LEVEL_DIRECTORIES.map { |dir| HOMEBREW_PREFIX/dir }
+  ALL_SHARE_PATHS = SHARE_PATHS.map { |dir| HOMEBREW_PREFIX/"share"/dir }
+  ALL_TOP_AND_SHARE_PATHS = ALL_TOP_LEVEL_PATHS + ALL_SHARE_PATHS
```

Comment:
I think these should all live in `diagnostic.rb` as they aren't used in `keg.rb`